apis/betterself/v1/correlations/views.py

```diff
@@ -101,3 +104,36 @@ def get(self, request):
         filtered_correlation_series = filtered_correlation_series.sort_values(ascending=False)
 
         return get_sorted_response(filtered_correlation_series)
+
+
+class ProductivityUserActivitiesCorrelationView(APIView):
```

Comment:
Just refactor the names later, otherwise you'll never finish. I mean, it's 1AM and you're talking to yourself in a pull request.